templates/base.html

```diff
@@ -1,152 +1,129 @@
-        <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
-        <link href='https://fonts.googleapis.com/css?family=Oswald:400,700&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
-        {% bootstrap_css %}
-        <link rel="stylesheet" href="{% static "style/style.css" %}" type="text/css">
-        <link rel="stylesheet" href="{% static "style/sticky-footer.css" %}" type="text/css">
-        {% block extra_css %}
-        {% endblock %}
-        
-        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-        <script type="text/javascript"
-                src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
-        {% bootstrap_javascript %}
-        {% block extra_js %}
-        {% endblock %}
 
-        <script type="text/javascript">
-            {% block script %}{% endblock %}
-        </script>
+    {# CSS #}
+    <link rel="stylesheet" href="{% static "style/bootstrap/bootstrap.css" %}" type="text/css">
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+    <link href='https://fonts.googleapis.com/css?family=Oswald:400,700&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+    {% bootstrap_css %}
+    <link rel="stylesheet" href="{% static "style/style.css" %}" type="text/css">
+{#    <link rel="stylesheet" href="{% static "style/sticky-footer.css" %}" type="text/css">#}
+    {% block extra_css %}
+    {% endblock %}
 
-        {% block head %}{% endblock %}
-        {% block extra_head %}{% endblock %}
+    {# JS #}
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script type="text/javascript"
+            src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
+    {% bootstrap_javascript %}
+    {% block extra_js %}
+    {% endblock %}
 
-        <title>{% block title %}{% endblock %}</title>
-    </head>
-<body>
+    <script type="text/javascript">
+        {% block script %}{% endblock %}
+    </script>
+
+    {# Other <head> content #}
+    {% block extra_head %}{% endblock %}
 
-<!-- hide header to show only login page -->
-{% if user.is_authenticated %}
+    <title>{% block title %}{% endblock %}</title>
+</head>
 
+<body>
+{# do not show navbar on login page #}
+{% url 'userena_signin' as signin_url %}
+{% if signin_url not in request.path  %}
     <nav class="navbar navbar-default" role="navigation">
         <div class="container-fluid">
             <div class="navbar-header">
-                <a class="navbar-brand" href="{% url "latest" %}">Pathagar</a>
+                <a class="navbar-brand" href="{% url "latest" %}">
+                    {{ request.site.name }}</a>
```

Comment:
On a very smaller side note: as you probably noticed, I tend not to enforce line length limits on the `.html` file - basically because HTML tend to have a lot of long lines inevitably, and personally feel more convenient to be able to just view the document structure based on the indenting, thus avoiding breaks based on line length (unless really huge lines!).